pro/src/new_components/ConfirmDialog/ConfirmDialog.tsx

```diff
@@ -40,7 +42,7 @@ const ConfirmDialog = ({
       labelledBy={title}
       onDismiss={onCancel}
     >
-      <Icon className={styles['confirm-dialog-icon']} />
+      {!hideIcon && <Icon className={styles['confirm-dialog-icon']} />}
```

Comment:
La propriété icon du composant étant nullable, on pourrait se passer ici du boolean pour écrire :
```suggestion
      {icon && <Icon className={styles['confirm-dialog-icon']} />}
```

Reply:
On a une icone par défaut quand le composant est null (ligne 35), c'est pour ça j'ai ajouter ce booléan.

Reply:
Yes au temps pour moi j'étais passé à côté de cette ligne, c'est tout bon :)